### PR TITLE
chore: added message for successful initialization

### LINKED ISSUE #1400: Missing Success Message Upon Initialization Completion
### Description

This RFC proposes :

Upon successful initialization by the `make init` command, nothing is displayed on the console. Error messages are displayed only when there are errors. If everything goes right, nothing is displayed. This might make new contributors a bit confused, whether they are on the right track to set up the project or not.

### Related Issues

Related issues # , # , and # .

### Questions

No.

### Other

A success message can be displayed once the initialization is successful.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [ ] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by gunjjoshi**
Added a success message in the Makefile, which shows up on successful initialization

Resolves #1400 

## Description

> What is the purpose of this pull request?

This pull request:

-   adds "Initialization Successful." message after successful execution of `make init`.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #1400 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

Although a small task, but still it can give clarity to new contributors.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [ ☑️] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Review by Planeshifter — APPROVED** [review]

**Comment by Planeshifter** [comment]
That should work; thanks for adding!

File: tools/make/lib/init/Makefile [hunk 1]
```diff
@@ -36,5 +36,6 @@ include $(TOOLS_MAKE_LIB_DIR)/init/git_hooks.mk
 # make init
 #/
 init: init-git-hooks init-eslint-plugins
+	@echo "Initialization Successful."
```
Comment (kgryte) [inline tools/make/lib/init/Makefile:39]: @Planeshifter This should be `$(QUIET) echo "..."`

Comment (kgryte) [reply tools/make/lib/init/Makefile:39]: I addressed in a follow-up commit.

Comment (gunjjoshi) [reply tools/make/lib/init/Makefile:39]: Thanks for correcting.

**Pull request merged**